eratos_xarray/backend/eratos_.py

```diff
@@ -67,35 +71,26 @@ def _raw_indexing_method(self, key: tuple) -> np.typing.ArrayLike:
         # If the returned array hdoes not have the expected number of dimensions, then remove integer selected dimensions out.
         if (array.ndim != self.ndim - len(axis)) and axis:
             array = np.squeeze(array, axis)
```

Comment:
My only question here is what about this code path and the dimension squeeze? I vaguely remember having to add this to reduce dimension sizes in some cases. Is it true that it's not relevant for the early return at line 64 since. Is there a case where the shape could have one zero length dimension, but also other 1-length dimensions which should be reduced?

This code was originally based off the pydap backend. I haven't been able to find an authoritative reference of how the backend _should_ behave in this regard.

Reply:
From what I understand this only gets used if we're selecting by raw indices or values rather than using `slice`, even if `slice` returns a 1-length dimension it still keeps it. Note [here](https://docs.xarray.dev/en/stable/user-guide/indexing.html#dataset-indexing) the difference between, say
```
ds.isel(time=0)
```
and 
```
da.isel(time=[0])
```
I believe the former results in the dimension squeeze.

Reply:
> Generally looks Ok. Just i have that question about the dimension reduction. Also do you have a preference for double quotes? any reason for changing all the quotes from single to double?

I'm not sure why but I think my editor just changed all of them.. generally I do prefer double quotes yes but I realised it made the diff a lot larger than it needs to be.

Reply:
Ok, that makes sense i think.

Reply:
Most formatters will change single quotes to double quotes with the exception of using single and double quotes in conjunction for a single string. e.g 
```python
print(f"my dict variable {value.get('key')}")
```